permissionsHandler.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	yaml "gopkg.in/yaml.v2"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/alexellis/derek/types"
+)
+
+const maintainersFileEnv = "maintainers_file"
+const defaultMaintFile = ".DEREK.yml"
+
+func getEnv(envVar string, assumed string) string {
```

Comment:
can assumed be `default`? Like os.getenv in Python?

Reply:
moot point given we're removing `maintainersFileEnv`

Tried `default` prior to assumed but it was linting as a reserved word.